syntax = "proto3";

package fastmitigation;

service Fastnetmon {
    rpc GetBanlist(BanListRequest) returns (stream BanListReply) {}
}

// We could not create RPC method without params
message BanListRequest {

}

message BanListReply {
    string ip_address = 1;
}
